app/routines/search_users.rb

```diff
@@ -83,9 +83,11 @@ def exec(query, options={})
       end
 
       with.keyword :uuid do |uuids|
-        sanitized_uuids = sanitize_strings(uuids, append_wildcard: options[:admin],
-                                            prepend_wildcard: options[:admin])
-        users = users.where{uuid.like_any sanitized_uuids}
+        if not uuids[0].include? '-' and uuids[0].to_s.length == 8
```

Comment:
Can you replace this condition with `if uuids.size == 1 and not (partial_uuid = uuids[0].to_s.chomp('-')).include? '-' and partial_uuid.length == 8`? (and use the partial_uuid variable below instead of uuids[0])

Just a minor thing to make it not have the `between` behavior for arrays with more than 1 element and to ignore `-` at the end of the uuid.

Reply:
This might work better actually:

```
uuids_queries = uuids.map do |uuid|
  partial_uuid = uuid.to_s.chomp('-')
  next uuid if partial_uuid.include? '-' or partial_uuid.length != 8

  partial_uuid + '-0000-0000-0000-000000000000'..partial_uuid + '-ffff-ffff-ffff-ffffffffffff'
end
users = users.where(uuid: uuids_queries)
```